charts/artifacts/templates/deployment.yaml

```diff
@@ -46,20 +46,12 @@ spec:
           httpGet:
             path: /_healthz
             port: http
-          initialDelaySeconds: 5
-          periodSeconds: 5
-          timeoutSeconds: 5
-          successThreshold: 1
-          failureThreshold: 3
+{{ toYaml .Values.deployment.artifacts.readinessProbe | indent 10 }}
         livenessProbe:
           httpGet:
             path: /_healthz
             port: http
-          initialDelaySeconds: 5
-          periodSeconds: 60
-          timeoutSeconds: 5
-          successThreshold: 1
-          failureThreshold: 3
+{{ toYaml .Values.deployment.artifacts.livenessProbe | indent 10 }}
```

Comment:
```suggestion
{{ toYaml .Values.deployment.artifacts.livenessProbe | indent 4 }}
```

Reply:
Same story, setting it to 4 will break the service.